Source/Scene/Implicit3DTileContent.js

```diff
@@ -10,6 +10,7 @@ import Rectangle from "../Core/Rectangle.js";
 import when from "../ThirdParty/when.js";
 import ImplicitSubtree from "./ImplicitSubtree.js";
 import ImplicitTileMetadata from "./ImplicitTileMetadata.js";
+import parseBoundingVolumeSemantics from "./parseBoundingVolumeSemantics.js";
```

Comment:
Make sure to update the semantics in `MetadataSemantic` (mainly for @IanLilleyT's usage)